doc/changes.rst

```diff
@@ -1,6 +1,111 @@
 Changelog
 =========
 
+Release v0.3.0 (2021/02/15)
+---------------------------
+
+.. image:: https://zenodo.org/badge/DOI/10.5281/zenodo.4522136.svg
+   :alt: Digital Object Identifier for the Zenodo archive
+   :target: https://doi.org/10.5281/zenodo.4522136
+
+Highlights
+
+*
+
+New Features
+
+* Wrap grd2cpt (#803), inset (#788), and subplot (#822)
+* Let Figure.text() support record-by-record transparency (#716)
+* Provide basic support for FreeBSD (#700)
+
+Enhancements
+
+* Let load_earth_relief() support 'region' argument for all resolutions (#873)
+* Detect ghostscript and default image viewer in FreeBSD (#878)
+* Improve how PyGMT finds the GMT library (#702)
+* Add dcw (#765) and lakes (#781) aliases to Figure.coast
+* Add the shading parameter to colorbar() in base_plotting.py (#752)
+
+Documentation
+
+* Update the docstrings in the plotting modules (#881)
+* Update the doc strings in the non-plotting modules (#882)
+* Add alias annotation (A) to Figure.contour() (#883)
+* Revise pygmt development installation instructions (#865)
+* Update the docstrings of common aliases (#862)
+* Add common alias panel (-c) to all plotting functions (#853)
+* Update forum link to pygmt specific category (#858)
+* Wrap grdinfo aliases (#799)
+* Add sphinx-copybutton extension to easily copy codes (#838)
+* Add additional video link in text for clarity (#836)
```

Comment:
```suggestion
```

Reply:
Same, I'll add a skip-changelog label to this PR (nothing against Liam).